Exercise-1/E1-TheBasicsOfHTML.html

```diff
@@ -1,17 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
+        <meta charset="utf-8" />
         <title>Exercise 1</title>
     </head>
     <body>
-        <h1>My todo list</h1>   
-        <ul>
-            <li>Make breakfast</li>
-            <li>Go to the supermarket</li>
-            <li>Walk the dog</li>
-            <li>Study</li>
-            <li>Go to class</li>
-            <li>Clean the house</li>                        
-        </ul>
+        <section>
+            <article>
+                <h1>My todo list</h1>
+                    <ul>
```

Comment:
The identation of the `ul` as also the `li` might be different to be consistent. But we will talk about that today.